src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+
 public class Duke {
+    private static Task[] items = new Task[100];
+    private static int taskCount = 0;
+    private static String logo = " _____  ___ _____\n"
+            + "|___  | | ||_____| \n"
+            + "   / /  | |   / / \n"
+            + "  / /   | |  / /  \n"
+            + " / /___ | | /_/__  \n"
+            + "|_____| | ||_____| \n";
+    private static String border = "____________________________________________________________\n";
+
+    public static void addedTaskMessage(Task task) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task);
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+    }
+    public static void addEvent(String description, String time) {
+        Events newEvent = new Events(description, time);
+        items[taskCount] = newEvent;
+        taskCount++;
+        addedTaskMessage(newEvent);
+    }
+    public static void addDeadline(String description, String by) {
+        Deadlines newDeadline = new Deadlines(description, by);
+        items[taskCount] = newDeadline;
+        taskCount++;
+        addedTaskMessage(newDeadline);
+    }
+    public static void addTodo(String description) {
+        Todo newTodo = new Todo(description);
+        items[taskCount] = newTodo;
+        taskCount++;
+        addedTaskMessage(newTodo);
+    }
+    public static void addTask(String description) {
+        System.out.println(border + "added: " + description + '\n' + border);
+        Task newItem = new Task(description);
+        items[taskCount] = newItem;
+        taskCount++;
+    }
+    public static void printStartMessage() {
+        System.out.println(logo);
+        System.out.println(border + "Hi bro, my name is Echo");
+        System.out.println("What do you want?\n" + border);
+        System.out.println("Type bye to exit\n" + border);
+    }
+    public static void printEndMessage() {
+        System.out.println(border);
+        System.out.println("chat again next time!\n" + border);
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        printStartMessage();
+
+        do {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            if (line.matches("list")) {
+                int j = 1;
+                System.out.println(border);
+                System.out.println("Here is your list");
+
+                for (Task item : items) {
+                    if (item != null) {
+                        System.out.print(j + ".");
+                        System.out.println(item);
+                        j++;
+                    }
+                }
```

Comment:
can consider moving this to a new method, might look neater, and no need so many level of indentation